modules/ROOT/pages/how-to-export-resources.adoc

```diff
@@ -5,10 +5,12 @@ endif::[]
 
 Mule Applications, Mule Domains and Mule Policies contain a set of resources which may be properties files, DataWeave transformation files, Java classes, etc. By default, all of these files are internal to the artifact in which they are defined, and as a result, they are inaccessible from other artifacts. For instance, if a resource defined at the domain level is not explicitly exported, then it cannot be accessed from within any of the applications that belong to that shared domain. Another example is classes from an application that are meant to be used by the Java module. If those classes are not exported, then the Java module will not be able to access them.
 
-== How to export Java classes
+== How to Export Java Classes
 
 To export Java classes, use the `exportedPackages` field of the `mule-artifact.json` file of your artifact. Keep in mind that only complete packages can be exported and not individual classes.
```

Comment:
Although it wasn't changed in this edit, I would start by saying that you can't export individual classes and that if you need to export classes, you must export the package containing them. (Assuming I understood this correctly).

I make this comment because of one of the considerations below.